FLIR/conservator/managers/__init__.py

```diff
@@ -122,8 +121,9 @@ def upload(
 
         if media:
             logger.info("Uploading media to collection %s", collection.path)
-            self._conservator.upload_many_to_collection(
-                media_paths, collection, max_retries=max_retries
+            media_manager = MediaTypeManager(self._conservator)
```

Comment:
Can we add `MediaTypeManager` instance to `Conservator` in `__init__`?
For instance, `self.media = MediaTypeManager(self)`, so here you could do `media_manager = self._conservator.media`

Reply:
Hmm, I suppose `conservator.videos` and `images` are already `MediaTypeManager`, so maybe reuse one of those for now

Reply:
I thought about having a default instance for media manager, but media is not a full-featured searchable graphql-query-backed structure like collection, dataset, video, or image, just a place to put common code supporting video and image. So I think API users would be disappointed to see this media instance and find that it is really not like the others, kind of ugly API design to imply it is similar. For that reason I'd rather use `videos` instance for upload.

Reply:
Makes sense. I think eventually we should create a "fully-featured searchable" manager that combines Video and Image paginated queries, and move a lot of this over there. 

We can also discuss with conservator folks if it's worth making new GraphQL Media queries that combine Videos and Images (shouldn't be hard. internally they're stored in the same db and filtered by `frameCount`). The conservator team has said they're willing to expand/create GraphQL queries for use cases outside of frontend UI

Reply:
Well that would be nice for API users to have built-in support for media in Conservator! Yes let's mention it to them.